packages/nextjs/src/adapter-nextjs/page-props/story.ts

```diff
@@ -16,14 +17,15 @@ import type { PropsFunction } from './lib/types';
 
 export function getStoryPageServerSideProps<CustomProps extends Record<string, any>>(
     customProps: CustomProps | PropsFunction<CustomProps>,
+    formats?: Story.FormatVersion[],
```

Comment:
Do we need to do the same for `getStoryPageStaticProps()` below and `getStoryPreviewPageServerSideProps()` in the neighbouring storyPreview.ts file?

Reply:
Yeah agree, making the update

Reply:
Updated